wikipedia_tool.py

```
@@ -0,0 +1,255 @@
+
+    params = {
+        "action": "query",
+        "titles": title,
+        "prop": "images",
+        "format": "json"
+    }
+
+    try:
+        res = requests.get(url, params=params, timeout=5)
+        data = res.json()
+    except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)